test/routes/test_wikiSaveRoute.ts

```
@@ -0,0 +1,169 @@
+    const original = "---\ntitle: Foo\ntags: [a, b]\n---\n\n- [ ] task one\n- [ ] task two\n";
+    await writeFile(filePath, original, "utf-8");
+
+    const updated = "---\ntitle: Foo\ntags: [a, b]\n---\n\n- [x] task one\n- [ ] task two\n";
+    const { state } = mockRes();
+    const { res } = mockRes();
+    await postWikiHandler(req({ action: "save", pageName: slug, content: updated }), res);
+
+    const onDisk = await readFile(filePath, "utf-8");
+    assert.equal(onDisk, updated);
+    assert.match(onDisk, /^---\ntitle: Foo/, "frontmatter delimiters should round-trip");
+    void state;
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)